### DISCUSSION TITLE
Multiple scroll wheen events per click

### DISCUSSION THREAD

**Discussion by bpershall9962** [hyprwm/Hyprland#6894] [Q&A] [answered]
I've run into an issue lately where scrolling with my Logitech wireless mouse results in multiple scroll wheel event per click. I was wondering if anyone had seen this before or was able to find a resolution to it? From my reading online, I haven't really been able to find anything significant on this nor have I been able to find a resolution. On my G502 I can plug it in to charge and the mouse wheel suddenly starts behaving properly so the issue only seems pertinent on wireless mice.

As a quick note, I've already tried adjusting the scroll factor in Hyprland with no luck.

**Comment by Agent00Ming** [comment, marked as answer]
You're welcome to try out #6881, user reports seem to indicate that it works pretty well. The issue you are bringing up is tracked as #6681.

**Reply by bpershall9962** [reply]
Awesome, I wasn't sure if this was a bug in libinput or Hyprland but I'm at ease knowing that it's being tracked. Much appreciated!
